centos7/Dockerfile.pgo-backrest-repo.centos7

```diff
@@ -7,19 +7,14 @@ LABEL name="pgo-backrest-repo" \
 	summary="Crunchy PostgreSQL Operator - pgBackRest Repository" \
 	description="Crunchy PostgreSQL Operator - pgBackRest Repository"
 
-ENV PGVERSION="12" PGDG_REPO="pgdg-redhat-repo-latest.noarch.rpm" PGDG_REPO_DISABLE="pgdg*" PGDG_REPO_ENABLE="pgdg12"\
-    BACKREST_VERSION="2.20"
-
-# PGDG PostgreSQL Repository
-RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/${PGVERSION}/redhat/rhel-7-x86_64/${PGDG_REPO}
+ENV PGVERSION="12" BACKREST_VERSION="2.20"
```

Comment:
I think we want to get the `12` into the Makefile (separate commit)...likely to `2.20` as well (separate commit)